src/vivarium_nih_us_cvd/components/healthcare_visits.py

```diff
@@ -0,0 +1,266 @@
+from typing import Dict
+
+import pandas as pd
+from vivarium.framework.engine import Builder
+from vivarium.framework.event import Event
+from vivarium.framework.population import SimulantData
+
+from vivarium_nih_us_cvd.constants import data_keys, data_values, models
+
+FOLLOWUP_MIN = 3 * 30  # 3 months
```

Comment:
is it standard in this project to have constants that are here vs in data_values, metadata, or within the class? i haven't seen this before and can't think of a reason it would be a problem, but it feels slightly messier

Reply:
No, you're probably right - I'll move those into data_values